sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/RetryOptions.java

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp;
+
+import java.time.Duration;
+import java.util.Objects;
+
+/**
+ * A set of options that can be specified to influence how retry attempts are made.
+ */
+public class RetryOptions implements Cloneable {
+    private int maxRetries;
+    private Duration delay;
+    private Duration maxDelay;
+    private Duration tryTimeout;
+    private RetryMode retryMode;
+
+    /**
+     * Creates an instance with the default retry options set.
+     */
+    public RetryOptions() {
+        maxRetries = 3;
```

Comment:
how did we decide on the defaults?

Reply:
They were the existing ones from the Track 1 library. And then I copied the ones @jsquire had in his implementation. :)